src/schematools/contrib/django/management/commands/change_dataset.py

```diff
@@ -1,21 +1,23 @@
+import argparse
 from argparse import ArgumentTypeError
 from distutils.util import strtobool
+from typing import Any
 
 from django.core.exceptions import ValidationError
 from django.core.management import BaseCommand, CommandError
 
 from schematools.contrib.django.models import Dataset
 
 
-def _strtobool(value):
+def _strtobool(value: Any) -> bool:
     try:
         return bool(strtobool(value))
     except ValueError:
         raise ArgumentTypeError("expected boolean value") from None
```

Comment:
I realize this is existing code, but the message is incorrect. A boolean value was NOT expected. On the contrary the boolean value is what this function is supposed to be generating. An arguably better message would be: _Expected a truth value_. Which is basically the message of the `ValueError` exception. Hence my suggestion would be to change this into:

```python
except ValueError as ve:
            raise ArgumentTypeError(str(ve)) from None
```